### PR TITLE
Update WASM link

### PR DESCRIPTION
**Pull request by ggsharma**
WasmFiddle link was broken. I created a brand new website (wasmgen.vercel.app) that compiles cpp code to JS and wasm using emscripten and thought it might be useful to share it with the users

### Description

Just replaced an unbroken link with a new link. This is one of the projects I recently worked on.

### Motivation

When I was starting learning about webassembly, I stumbled upon this page. Some of the links to an online wasm generator didn't work and the one's that did, didn't give any valid HTTP response. I thought, it might be a good opportunity for me to learn about web assembly by making an online C++ to WASM compiler.

### CODE REVIEW

File: files/en-us/webassembly/concepts/index.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Let's talk about these options:
 
 Two of the many options for creating Wasm code are an online Wasm assembler or [Emscripten](https://emscripten.org/). There are a number of online Wasm assembler choices, such as:
 
-- [WasmFiddle](https://wasdk.github.io/WasmFiddle/)
+- [WasmGen](https://wasmgen.vercel.app)
```
Comment (bsmth) [inline files/en-us/webassembly/concepts/index.md:82]:
```suggestion
```

Thanks a lot for opening a pull request, however the contribution guidelines strongly [discourage submitting links to your own projects](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#external_links) so in this case we have to remove the existing link only. Sorry if it's disappointing 🙏🏻

**Comment by gautam-sharma1** [comment]
Removed the broken link

**Comment by bsmth** [comment]
> Removed the broken link

Thank you! And well done on your first PR here, welcome aboard!

**Review by bsmth — APPROVED** [review]

**Comment by gautam-sharma1** [comment]
Thank you for the encouragment Brian 😀

**Pull request merged**